### PR TITLE
auth-validation

### PR DESCRIPTION
**Pull request by AlexandrKlychnikov**
implement authentication user and validation login and signup forms with tips

### CODE REVIEW

**Review by aibolit666 — APPROVED** [review]
Валидация логина отлично сделана

File: src/App.tsx [hunk 1]
```diff
@@ -72,7 +73,7 @@ function App() {
           <Footer />
         </>
       ) : (
-        <CircularProgress />
+        <CircularProgress sx={{ position: 'absolute', top: '50%', left: '50%' }} />
```
Comment (MatsveiDubaleka) [inline src/App.tsx:76]: Yeap, when it in the center looks great, then at the corner

File: src/api/registerService.ts [hunk 2]
```diff
@@ -54,16 +54,21 @@ export async function createUser(newuser: IFormInputs): Promise<INewUserResponse
 }
 
 export async function loginUser(user: IFormInputs): Promise<IAuthorizationResult | undefined> {
+  const response = await fetch(`${API_URL}${Endpoint.SIGN_IN}`, {
```
Comment (MatsveiDubaleka) [inline src/api/registerService.ts:57]: I told Pasha about this, we have axios, we dont need to use fetch

File: src/components/Header.tsx [hunk 3]
```diff
@@ -37,7 +37,7 @@ function Header() {
 
   const handleClickLogOut = () => {
     navigate(Navigation.HOME);
-    dispatch(setToken(''));
+    dispatch(setValidation(false));
```
Comment (MatsveiDubaleka) [inline src/components/Header.tsx:40]: Looks more logical

**Review by MatsveiDubaleka — APPROVED** [review]
Good job

**Pull request merged**